src/components/video-editor/settings/sections/ExtensionSettingsSection.tsx

```
@@ -0,0 +1,227 @@
+									forceUpdate((n) => n + 1);
+								}}
+								className="data-[state=checked]:bg-[#2563EB] scale-75"
+							/>
+						</div>
+					);
+				}
+
+				if (field.type === "slider") {
+					const step = field.step ?? 0.01;
+					// Basic memoization for precision calculation
+					const precision = useMemo(() => getStepPrecision(step), [step]);
```

Biome (2.4.15)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

(lint/correctness/useHookAtTopLevel)